client/web/compose/src/views/Admin/Pages/Edit.vue

```diff
@@ -1021,6 +1011,10 @@ export default {
         off: this.$t('general:label.no'),
       },
 
+      tableWrapperLabel: {
```

Comment:
Not really good practice to name stuff after component.
It would have to be named by the context where its used (layouts in this case)
I honestly prefer inline for labels (translations). if it's not used multiple times.
Only logical code needs to be inside the <script>, or if its used multiple times